src/GafferUI/StandardNodeGadget.cpp

```diff
@@ -400,9 +400,24 @@ void StandardNodeGadget::doRenderLayer( Layer layer, const Style *style ) const
 				m_userColor.get_ptr()
 			);
 
-			if( MetadataAlgo::getBookmarked( node() ) )
+			int value = MetadataAlgo::getNumericBookmark( node() );
+			// \todo: should we support showing larger values?
+			// \todo: should adding a numeric bookmark set the metadata for 'normal' bookmarks? that would simplify the following
```

Comment:
I think this is a bit problematic, because we won't know when to remove the "normal" bookmark. If the node wasn't bookmarked normally beforehand, we should be removing the normal bookmark when the numeric one is removed. But if it was bookmarked normally beforehand, we want it to stay.

Reply:
Perhaps we should adjust the drawing slightly to show the different states :

- For a regular bookmark, draw a yellow star
- For a numeric bookmark only, draw a grey star with the number overlaid
- For both, draw a yellow star with the number overlaid